include/openenclave/bits/fs.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_BITS_FS_H
+#define _OE_BITS_FS_H
+
+/*
+**==============================================================================
+**
+** This file defines functions for loading internal modules that are part of
+** the Open Enclave core.
+**
+**==============================================================================
+*/
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+#include <openenclave/bits/types.h>
+
+OE_EXTERNC_BEGIN
+
+/**
+ * Name of the non-secure host file system (passed to **mount()** as the
+ * **filesystemtype** parameter).
+ */
+#define OE_HOST_FILE_SYSTEM "oe_host_file_system"
```

Comment:
>OE_HOST_FILE_SYSTEM [](start = 8, length = 19)

For my understanding, why do these need to be in bits/ unlike the others declared in the internal/posix/device.h? Is this a precursor to the next PR for filesystem?

Reply:
Yes, that is correct. All devices have names and you will see this in next PR.